server/region_handler.go

```diff
@@ -227,6 +233,53 @@ func (t *regionHandlerTool) getRegionsMeta(regionIDs []uint64) ([]RegionMeta, er
 	return regions, nil
 }
 
+// ServeHTTP handles request of list a database or table's schemas.
+func (rh schemaHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	schema, err := rh.schema()
+	if err != nil {
+		rh.writeError(w, err)
+		return
+	}
+
+	// parse params
+	params := mux.Vars(req)
+
+	if dbName, ok := params[pDBName]; ok {
+		cDBName := model.NewCIStr(dbName)
+		if tableName, ok := params[pTableName]; ok {
+			// table schema of a specified table name
+			cTableName := model.NewCIStr(tableName)
+			if data, err := schema.TableByName(cDBName, cTableName); err != nil {
+				rh.writeError(w, err)
```

Comment:
Could we `return` after `writeXXX`, since we do not care about this logic anymore?

Reply:
Done!